core/src/main/resources/hudson/tasks/Shell/help-unstableReturn.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    If set, the shell exit code that will be interpreted as an unstable build result. On Unix-like systems the value between 0-255.
```

Comment:
How about: 
“If set, the shell exit code that will be interpreted as an unstable build result. If the exit code matches the value, the build results will be set to 'unstable' and next steps will be continued. On Unix-like it is a value between 0-255.”